lib/Controller/LocalController.php

```diff
@@ -239,6 +239,7 @@ public function streamNotifications($since = 0, int $limit = 5): DataResponse {
 	/**
 	 * @NoAdminRequired
 	 * @NoSubAdminRequired
+	 * @PublicPage
```

Comment:
This allows guest users to fetch the account stream.

Reply:
not sure we want this. We already have a streamAccount ?

Reply:
This is actually just adding the PublicPage annotation to the streamAccount method (GitHub diff is a bit strange :wink: )